core/src/main/scala/cats/instances/duration.scala

```diff
@@ -1,9 +1,28 @@
 package cats
 package instances
 
-import scala.concurrent.duration.Duration
+import scala.concurrent.duration.{Duration, FiniteDuration}
 
 trait DurationInstances extends cats.kernel.instances.DurationInstances {
-  implicit val catsStdShowForDuration: Show[Duration] =
+
+  @deprecated("Left to keep binary compatibility. Use CoreDurationInstances.catsStdShowForDurationUnambiguous instead.",
+              "1.5.0")
+  val catsStdShowForDuration: Show[Duration] =
+    AllCoreDurationInstances.materialise.catsStdShowForDurationUnambiguous
+}
+
+private[instances] trait AllCoreDurationInstances extends CoreFiniteDurationInstances
+
+private[instances] trait CoreFiniteDurationInstances extends CoreDurationInstances {
+  implicit final val catsStdShowForFiniteDurationUnambiguous: Show[FiniteDuration] =
+    Show.fromToString[FiniteDuration]
+}
+
+private[instances] trait CoreDurationInstances {
+  implicit final val catsStdShowForDurationUnambiguous: Show[Duration] =
     Show.fromToString[Duration]
 }
+
+private[instances] object AllCoreDurationInstances {
```

Comment:
nitpick here, but can't we just let `object AllCoreDurationInstances` extend `trait AllCoreDurationInstances`? That would be more conventional than having a `materialise` instance right?  I could be missing something.

Reply:
Very true. I don't remember from which phase it has remained, it's definitely not needed. Changing it.